### PR TITLE
Read scaffolding config from yaml file

### PR DESCRIPTION
**Pull request by boma96**
Things done:

- Integrated command line parser library into tool
- Added library for parsing yaml files
- Scaffolding configuration is now specified in yaml file

### CODE REVIEW

File: src/ConductorSharp.Toolkit/Program.cs [hunk 1]
```diff
@@ -4,72 +4,125 @@
...
+                .Deserialize<Configuration>(File.ReadAllText(configFilePath));
 
-        private static IContainer BuildContainer(CommandInput input)
+        private static bool ValidateConfiguration(Configuration config)
+        {
+            if (config.BaseUrl == null)
+                Console.Error.WriteLine("baseUrl property missing in configuration");
+            if (config.Namespace == null)
+                Console.Error.WriteLine("namespace property missing in configuration");
+            if (config.Destination == null)
```
Comment (ognjenkatic) [inline src/ConductorSharp.Toolkit/Program.cs:89]: We should probably check for null or empty here just in case

**Review by ognjenkatic — COMMENTED** [review]
LGTM, just one minor suggestion

File: src/ConductorSharp.Toolkit/Program.cs (hunk 1 above)
Comment (boma96) [reply src/ConductorSharp.Toolkit/Program.cs:89]: Agree, fill fix it

Comment (boma96) [reply src/ConductorSharp.Toolkit/Program.cs]: Done

**Comment by ognjenkatic** [comment]
LGTM

**Review by ognjenkatic — APPROVED** [review]

**Pull request merged**